app/Models/Subject.php

```diff
@@ -8,7 +8,7 @@ class Subject extends Model
 {
     protected $tables = 'subjects';
     protected $fillable = [
-    	'id',
-    	'name'
+        'id',
+        'name'
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoCommaAfterLast</br>**Severity**: error</br>**File**: app/Models/Subject.php</br>**Line**: 12</br>Comma required after last value in array declaration